### ISSUE TITLE
Running kitchen verify asks for 'roots' password

### ISSUE THREAD

**Issue by aaronlippold** [dev-sec/ansible-nginx-hardening#11]
```
-----> Creating <nginx-ansible-latest-centos-7>...
       Creating kitchen sandbox at /Users/aaronl/.dokken/kitchen_sandbox/fea2d64b9d-nginx-ansible-latest-centos-7
       Creating verifier sandbox at /Users/aaronl/.dokken/verifier_sandbox/fea2d64b9d-nginx-ansible-latest-centos-7
       Building work image..
       Finished creating <nginx-ansible-latest-centos-7> (0m2.79s).
-----> Converging <nginx-ansible-latest-centos-7>...
       Creating kitchen sandbox in /Users/aaronl/.dokken/kitchen_sandbox/fea2d64b9d-nginx-ansible-latest-centos-7
       Preparing playbook
       Preparing inventory
       Preparing modules
       nothing to do for modules
       Preparing roles
       Preparing ansible.cfg file
       Empty ansible.cfg generated
       Preparing group_vars
       nothing to do for group_vars
       Preparing additional_copy_path
       Preparing host_vars
       nothing to do for host_vars
       Preparing hosts file
       Preparing spec
       Preparing library plugins
       nothing to do for library plugins
       Preparing callback plugins
       nothing to do for callback plugins
       Preparing filter_plugins
       nothing to do for filter_plugins
       Preparing lookup_plugins
       nothing to do for lookup_plugins
       Preparing additional_ssh_private_keys
       nothing to do for additional_ssh_private_keys
       Finished Preparing files for transfer
       Installing ansible using ansible omnibus
root@'s password:

...
```

**Comment by rndmh3ro** [comment]
That should be fixed once I get #8 into a usable state.

**Comment by rndmh3ro** [comment]
Now that #8 is merged, this should not be a problem anymore.
